### PR TITLE
update README for 1.4.0

### PR DESCRIPTION
**Pull request by kaakaa**
refs #328

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Matterpoll is a plugin for [Mattermost](https://mattermost.com/). It allows user
 ## Installation
 
 1. Go to the [releases page of this GitHub repository](https://github.com/matterpoll/matterpoll/releases/latest) and download the latest release for your Mattermost server.
-2. Upload this file in the Mattermost **System Console > Plugins > Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+2. Upload this file in the Mattermost **System Console > Plugins > Plugin Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#custom-plugins).
 3. You should set **Enable integrations to override usernames** and **Enable integrations to override profile picture icons** in **System Console > Custom Integrations** to `true`.
```
Comment (kaakaa) [inline README.md:16]: This settings are still needed?

Comment (hanzei) [reply README.md:16]: No, they should get removed.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Matterpoll is a plugin for [Mattermost](https://mattermost.com/). It allows user
 ## Installation
 
 1. Go to the [releases page of this GitHub repository](https://github.com/matterpoll/matterpoll/releases/latest) and download the latest release for your Mattermost server.
-2. Upload this file in the Mattermost **System Console > Plugins > Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+2. Upload this file in the Mattermost **System Console > Plugins > Plugin Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#custom-plugins).
```
Comment (hanzei) [inline README.md:15]: I hope we can simplify this to 
`In Mattermost, open **Main Menu > Plugin Marketplace** to install and configure Matterpoll.`

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for updating the docs :+1:

File: README.md [hunk 3]
```diff
@@ -21,20 +21,24 @@ You can configure Matterpoll from **System Console > Plugins > Matterpoll**.
...
+  - Change button color of voted answers
+  - Hide poll management buttons (Add Option / Delete Poll / End Poll) from users who don't have permission
 
 ## Usage
 
 `/poll "Is Matterpoll great?"` creates a poll with the answer options "Yes" and "No". You can also leave out the double quotes and just type `/poll Is Matterpoll great?`.
 
 If you want to define all answer options by yourself, type `/poll "Is Matterpoll great?" "Of course" "In any case" "Definitely"`- Note that the double quotes are required in this case.
 
+`/poll` show up interactive dialog for creating a poll.
```
[the remark this answers is not in the document]
Comment (kaakaa) [reply README.md:33]: Modal looks better for users. Thanks!

File: README.md (hunk 2 above)
Comment (kaakaa) [reply README.md:15]: Great 👍

**Comment by kaakaa** [comment]
Thanks for your some suggestions. I've fixed it. Please take a look again.

**Comment by kaakaa** [comment]
Added last one change for screenshot, because [community plugin steps](https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/#requirements-for-adding-a-community-plugin-to-the-marketplace) requires at least one screenshot.

> The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide.

File: README.md [hunk 4]
```diff
@@ -8,33 +8,36 @@
...
 
+![Matterpoll plugin screenshot](images/screenshot.png)
 
 ## Installation
 
 1. Go to the [releases page of this GitHub repository](https://github.com/matterpoll/matterpoll/releases/latest) and download the latest release for your Mattermost server.
-2. Upload this file in the Mattermost **System Console > Plugins > Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-3. You should set **Enable integrations to override usernames** and **Enable integrations to override profile picture icons** in **System Console > Custom Integrations** to `true`.
-
+2. In Mattermost, open **Main Menu > Plugin Marketplace** to install and configure Matterpoll.
```
Comment (hanzei) [inline README.md:16]: Sorry, my comment was ambiguous. I was suggesting to replace the whole "Installation" section with `In Mattermost, open **Main Menu > Plugin Marketplace** to install and configure Matterpoll.`

Comment (kaakaa) [reply README.md]: I missed that..🤦  Thanks!

**Review by hanzei — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 5]
```diff
@@ -8,33 +8,35 @@
...
 ## Installation
 
-1. Go to the [releases page of this GitHub repository](https://github.com/matterpoll/matterpoll/releases/latest) and download the latest release for your Mattermost server.
-2. Upload this file in the Mattermost **System Console > Plugins > Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-3. You should set **Enable integrations to override usernames** and **Enable integrations to override profile picture icons** in **System Console > Custom Integrations** to `true`.
-
+In Mattermost, open **Main Menu > Plugin Marketplace** to install and configure Matterpoll.
 
 ## Settings
 You can configure Matterpoll from **System Console > Plugins > Matterpoll**.
```
Comment (hanzei) [inline README.md:18]:
```suggestion
After the plugin has downloaded and been installed, click the **Configure** button. Alternatively, you can configure Matterpoll from **System Console > Plugins > Matterpoll**.
```

**Review by hanzei — COMMENTED** [review]
One suggestion from https://github.com/mattermost/matterpoll/pull/1#discussion_r520625576

**Pull request merged**